drupal/sites/all/modules/antistatique_article/src/blocks/warning.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function antistatique_article_warning_block($param = null) {
+    $variables = array();
+    global $language;
+
+    $node = menu_get_object();
+    if ($node = menu_get_object() && $node->language != $language->language) {
```

Comment:
Sorry fot the `menu_get_object()`  it came from test ...

Im'not sure. In fact the block should be displayed :
 - User in interface FR but in article English (no-translation) - Show
 - User in interface EN & in article French (no-translation) - Show
 - User in interface EN & in article English (original lang or translation) - Nothing
 - User in interface FR & in article French (original lang or translation) - Nothing

What do you think ?